cloud_functions/climateiq_data_export_cf/main.py

```diff
@@ -65,9 +65,13 @@ def export_model_predictions(cloud_event: http.CloudEvent) -> None:
         chunks_ref,
     )
 
-    # Set the error message to the Series' json string for testing
-    # purposes. In the actual implementation, the Series should be stored somewhere.
-    raise NotImplementedError(h3_predictions.to_json())
+    storage_client = storage.Client()
+    bucket = storage_client.bucket(bucket_name)
```

Comment:
these predictions should be written to another bucket. 

so far we have:
climateiq-predictions --> stores ML raw predictions
climateiq-chunk-predictions --> stores ML raw predictions per chunk (and is read by this cloud function)

how about climateiq-spatialized-chunk-predictions for outputs?